core/common/src/test/java/alluxio/ConfigurationTestUtils.java

```diff
@@ -75,6 +75,10 @@ public static InstancedConfiguration defaults() {
       conf.put(PropertyKey.Template.WORKER_TIERED_STORE_LEVEL_DIRS_PATH.format(level),
           Joiner.on(',').join(newPaths));
     }
+
+    // Sets up the block allocation and review policy
+    conf.put(PropertyKey.WORKER_REVIEWER_CLASS, "alluxio.worker.block.reviewer.AcceptingReviewer");
```

Comment:
This will help integration tests that use `defaults()` to avoid the non-deterministic block allocation reviewing